src/storage/config.js

```diff
@@ -0,0 +1,76 @@
+const db = require('../storage/DataAccess');
+const { CONFIG } = require('./collections');
+const { configVars } = require('./configVars');
+
+/**
+ * Fields in Config:
+ *
+ * Server:
+ *  @param {string} ADMIN_TOKEN - bearer token that grants access
+ *  @param {string} AUTH - auth base url
+ *  @param {string} AUTH_CERTS_URL: - auth url for certs retrieval
+ *  @param {string} AUTH_TOKEN_URL - auth url for token retrieval
+ *  @param {string} DATA_TRUST_SERVICE - the data trust service
+ *  @param {string} REALM - specific realm for the auth server
+ *
```

Comment:
Suggest adding REQUIRE_AUTH and REQUIRE_AUTH_FOR_OUTGOING_REQUESTS to be part of this config as well instead of being env